safety_gym/search/planner/planning_config.py

```diff
@@ -0,0 +1,28 @@
+planning_config = {
```

Comment:
This file is the configure file for the 2d_nav task. Why do we need it here?🤔

Reply:
Got it, thank you. That file is all defined now :)

Reply:
Did not see the commit yet... Did you forget to push?

Reply:
My apologies - the a* wasn't working until then. I have pushed it now (with a working a*) and you should be able to see it.